core/server/services/webhooks/trigger.js

```diff
@@ -0,0 +1,66 @@
+const _ = require('lodash');
```

Comment:
Now that `trigger` is part of the webhooks service, we can remove the impl  from the API, right?